fail2ban/client/fail2banregex.py

```diff
@@ -113,7 +113,9 @@ def format_help(self, *args, **kwargs):
 LOG:
     string                  a string representing a log line
     filename                path to a log file (/var/log/auth.log)
-    "systemd-journal"       search systemd journal (systemd-python required)
+    "systemd-journal"       search systemd journal. Optionally specify
+                            `systemd-journal[journalflags=X]` to determine
```

Comment:
Basically systemd backend has more parameters as just `journalflags`, see #1523 
Also I must admit, this is currently not documented in [man/jail.conf.5](https://github.com/fail2ban/fail2ban/blob/master/man/jail.conf.5#L300).
Would you possibly like to add that info also?

Reply:
Thanks for the feedback, see what you think.

Reply:
> you're not waiting on me, are you?

I thought we were in agreement about to extend this with short note about more parameters than just journalflags... or just a reference to jail.conf.5 which explaining now all that.
Although it is good enough too, so If you'd not extend that, I'll merge it later as it is.

Reply:
BTW, the notice "(systemd-python required)" rather belongs to "search systemd journal" as to the second sentence explaining options.

Reply:
Sorry, I may have misunderstood -- I thought you were asking me to add that info to `jail.conf.5`, so I did: https://github.com/fail2ban/fail2ban/pull/2809/commits/1b7cd75cc15c7f86f63f9c419de82b1b18291e0b